### PR TITLE
Display topic telephone infos in a box, on tab navigation

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

File: src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  transform: translate(-50%, 0);
+}
+
+.tm-w-m,
+.tm-w-s,
+.tm-w-xs {
+  .wkp-tel-infos {
+    top: 60px;
+  }
+}
```
Comment (oterral) [inline src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss]: Put them in trafimage-maps.scss with the others

Comment (edellucien) [reply src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss]: Didn't saw that we locate all screen size specific style in TrafimageMaps.scss now.
I updated it :)

**Comment by oterral** [comment]
@mhaertwig  code is fine for me

**Comment by edellucien** [comment]
I clarified the requirement with mario, I adapt the code and assign back to him.

**Comment by edellucien** [comment]
@oterral you can review the code again.

**Comment by oterral** [comment]
ok for me.

**Comment by edellucien** [comment]
Thanks olivier for adapting netlify config! :boom: :thumbsup: :trophy:
Doc test link: https://deploy-preview-285--trafimage-maps.netlify.com/
Build test link: https://deploy-preview-285--dev-trafimage-maps.netlify.com/

File: src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+.wkp-tel-infos {
+  position: absolute;
+  top: 110px;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 1;
+  height: 0;
+  background-color: white;
+  border: 0;
```
Comment (mhaertwig) [inline src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss]: @edellucien why do you set padding and height to "0"?

File: src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  padding: 0;
```
Comment (edellucien) [inline src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss]:
```suggestion
```

File: src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss (hunk 2 above)
Comment (edellucien) [reply src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss]:
```suggestion
```

Comment (edellucien) [reply src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss]: Indeed it is unecessary, setting it on focus is sufficient

**Pull request merged**